pkg/job/maxdimassociator/associator_directoryservice_test.go

```diff
@@ -0,0 +1,61 @@
+package maxdimassociator
```

Comment:
@RuslanMustaev sorry the license check is failing, could you please add the header to this file?

Reply:
Done